examples/basic/client/src/main.rs

```diff
@@ -6,56 +6,73 @@ use com::{
 use interface::{Food, IAnimal, ICat, IDomesticAnimal, IExample, CLSID_CAT_CLASS};
 
 fn main() {
+    // Initialize the COM apartment
     init_apartment(ApartmentType::SingleThreaded)
         .unwrap_or_else(|hr| panic!("Failed to initialize COM Library{:x}", hr));
+    println!("Initialized apartment");
 
+    // Get a `BritishShortHairCat` class factory
     let factory = get_class_object::<IClassFactory>(&CLSID_CAT_CLASS)
         .unwrap_or_else(|hr| panic!("Failed to get cat class object 0x{:x}", hr));
     println!("Got cat class object");
 
+    // Get an instance of a `BritishShortHairCat` as the `IUnknown` interface
```

Comment:
I assume this is just an example and not a requirement. 😉

Reply:
Do you mean you hope it's not a requirement to get the IUnknown interface? It's not as `get_instance` is generic over the interface you want to get the class as.

Reply:
Good, the example just made it look like I needed to retrieve the interface and then later query for the desired interface.